PJ_SIM_SUNBNB_2023-12 [FE] Home Page Card Component

ThereseRoca opened the pull request:
# Issue Link

[PJ_SIM_SUNBNB_2023-12 [FE] Home Page Card Component](https://framgiaph.backlog.com/view/PJ_SIM_SUNBNB_2023-12)

# Slack Link
https://sun-philippine.slack.com/archives/C05GURS7N3U/p1691147307571499

# Definition of Done
Create a card component for the listings containing:
- [x] Image
- [x] Label
- [x] Ratings
- [x] Reviews
- [x] Price
- [x] Type
- [x] Capacity

# Pre-conditions 
none

# Expected Output
![image](https://github.com/framgia/pj_sim-sunbnb/assets/138637515/19789e1a-7e95-4018-a44e-80a1d4f7749f)


# Screenshots
![image](https://github.com/framgia/pj_sim-sunbnb/assets/138637515/7f81ec50-800b-403c-8f2f-3ccfd60f27b8)

roseaugusto commented on .gitignore:
```diff
@@ -1,23 +1,25 @@
-*.njsproj
-*.sln
-*.sw?
+.DS_Store
```
what was change here?

ThereseRoca replied:
I accidentally put the login page before ma'am in .gitignore maam so I removed it there ma'am.

roseaugusto commented on package-lock.json:
```diff
@@ -0,0 +1,6 @@
+{
```
why is this change? there is no new package being installed so I think no need for this one to be modified

ThereseRoca replied:
It just seems to be there po ma'am even if I don't stage it for commit during that time.  It also occurred on some of the PR of my co-interns ma'am.

LennardUy104 replied:
Removed duplicate package-lock.json

LennardUy104 commented:
Resolve conflict and clean up code

LennardUy104 committed:
PJ_SIM_SUNBNB_2023-12 [FE] Home Page Card Component

fix listing component

remove extra package.json and fixed git ignore

Revert "PJ_SIM_SUNBNB_2023-12 [FE] Home Page Card Component"

This reverts commit 1882a2a51076dd1783ad13afdf371f26891adb6e.
